app/javascript/components/RequestCart.vue

```diff
@@ -0,0 +1,522 @@
+<template>
+
+  <transition name="slide">
+
+  <div v-if="isVisible" :class="['request-cart']">
+
+      <form method="post" :action="configuration.url">
+    <div v-if="requests.length" class="panel">
+        <table :class="['lux-data-table']">
+
+          <caption>
+
+    <input-button
+      v-on:button-clicked="toggleCartView($event)"
+      type="button"
+      variation="text"
+      class="denied-button"
+    >
+
+      <div class="lux-icon">
```

Comment:
This should use the same icon that is used in other RequestCart components. I noticed that there are both CartIcon and RequestCartIcon that have been added in this PR.